Source/NSDictionary+HYPImmutable.h

```diff
@@ -2,10 +2,12 @@
 
 @interface NSDictionary (HYPImmutable)
 
-- (NSDictionary *)hyp_dictionaryByRemovingNullItems;
+- (NSDictionary *)hyp_removingNulls;
 
-- (NSDictionary *)hyp_dictionaryByRemovingKey:(id <NSCopying>)key;
```

Comment:
Why not? Apple does it and I think it looks good

![screen shot 2015-03-31 at 5 42 16 pm](https://cloud.githubusercontent.com/assets/1088217/6922681/536c613a-d7cd-11e4-93e3-bd2ef13039a1.png)